support/req.sh

```diff
@@ -0,0 +1,4 @@
+curl -i -F "license_plates=BB111" -F "longitude=12222" -F "latitude=33333" -F "city=Kiev" -F "address=Foo-baar 33/3" \
```

Comment:
First of all it's useful hint for both back-end and mobile devs. Only then I consider it as test.